lib/conjur/fpm/Dockerfile

```diff
@@ -1,14 +1,14 @@
 # Build from the same version of ubuntu as phusion/baseimage
-FROM ubuntu:14.04
+FROM ubuntu:18.04
 
 RUN apt-get update -y && \
     apt-get install -y software-properties-common && \
     apt-add-repository -y ppa:brightbox/ruby-ng && \
     apt-get update -y && \
-    apt-get install -y build-essential git libpq5 libpq-dev ruby2.2 ruby2.2-dev libffi-dev
+    apt-get install -y build-essential git libpq5 libpq-dev ruby2.3 ruby2.3-dev libffi-dev
```

Comment:
was the change needed for ubuntu 18.04?

Reply:
Yes, because ruby2.2 isn't available in 18.04.